submissions/karlambuckner.md

```diff
@@ -0,0 +1,17 @@
+* submitter: @karlambuckner
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* @Portland is one of two cities in the US to have a dormant volcano within city limits.
+* @Portland holds an annual World Naked Bike Ride.
+* @Portland had the world's smallest park.
+* @Portland has more strip clubs per capita than any other city.
+* There's an underground tunnel in downtown @Portland.
+
+Karla:
+was born in Asheville, NC
```

Comment:
If you could add Markdown list stars (```*```) to each item in this list and indent everything by two spaces, that would be great! Other than that, perfect job!